Podcast/SearchITunesViewController.swift

```diff
@@ -16,7 +16,7 @@ class SearchITunesViewController: ViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .offWhite
+        view.backgroundColor = .paleGrey
         title = "Search iTunes"
         
         searchResultsController = SearchITunesTableViewController()
```

Comment:
is there a reason you did separate SearchITunesTableViewController and SearchITunesViewController? If not, we should actually combine these. You can say `searchResultsController = self` and make this view controller have a tableView within it